### ISSUE TITLE
bug: build problems when upDating hyperhtml

### ISSUE THREAD

**Issue by cousquer** [cousquer/CardWebComponents#37]
**Describe the bug**
The build fails when updating to hyperHTML to 2.16.1

**To Reproduce**
1. in package.json update hyperHTML to 2.16.1 

**Trace**

```
src/my-card.js → dist/my-card.esm.js, dist/my-card.umd.js, dist/my-card.cjs.js...
(!) `this` has been rewritten to `undefined`
https://rollupjs.org/guide/en#error-this-is-undefined
node_modules\@ungap\essential-weakset\esm\index.js
1: /*! (c) Andrea Giammarchi - ISC */
2: var self = this ||
              ^
3: /* istanbul ignore next */
4: {};
node_modules\@ungap\weakmap\esm\index.js
1: /*! (c) Andrea Giammarchi - ISC */
2: var self = this ||
              ^
3: /* istanbul ignore next */
4: {};
node_modules\@ungap\custom-event\esm\index.js
1: /*! (c) Andrea Giammarchi - ISC */
2: var self = this ||
              ^
3: /* istanbul ignore next */
4: {};
node_modules\@ungap\essential-map\esm\index.js
1: /*! (c) Andrea Giammarchi - ISC */
2: var self = this ||
              ^
3: /* istanbul ignore next */
4: {};
[!] (babel-minify plugin) Error: Error transforming bundle with 'babel-minify' plugin: don't know how to turn this value into a node
Error
```

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
me: 
![ryan](https://user-images.githubusercontent.com/1481430/49433386-329a1c00-f7b2-11e8-9945-eb0eda5d7733.gif)

**Comment by cousquer** [comment]
https://github.com/WebReflection/hyperHTML/issues/304

**Comment by cousquer** [comment]
resolved ✅  in #38
